src/components/play/PlayUI.js

```diff
@@ -51,7 +51,13 @@ class PlayUI extends Component {
         await this.createRound()
         window.addEventListener('resize', this.onWindowResizeThrottled)
         window.addEventListener('keypress', this.onKeypress)
-        window.addEventListener('dblclick', () => this.onMuteToggle(this.props))
+        window.addEventListener('dblclick', (e) => {
+            // should be a layer to mute toggle
+            const styleClass = e.target.classList[0]
+            if (styleClass && styleClass.indexOf('PlayUI-button-') > -1) {
```

Comment:
Sorta small, but you could use [includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes)